task-impls/src/consensus.rs

```diff
@@ -551,49 +830,83 @@ where
                                     .register_state(state)
                                     .register_event_handler(handle_event);
                             let id = builder.get_task_id().unwrap();
+
+                            self.vote_collector = Some((vote.current_view, id));
+
                             let _task = async_spawn(async move {
-                                VoteCollectionTypes::build(builder).launch().await
+                                VoteCollectionTypes::build(builder).launch().await;
                             });
-                            self.vote_collector = Some((vote.current_view, id));
+                            error!("Starting vote handle for view {:?}", vote.current_view);
                         }
                     }
                     QuorumVote::Timeout(_) | QuorumVote::No(_) => {
-                        warn!("The next leader has received an unexpected vote!");
+                        panic!("The next leader has received an unexpected vote!");
```

Comment:
I guess the idea is we are just removing no voting because it's just not implemented.  As a side note I wasn't sure which task view sync or consensus would handle forming the no quorum, but we can solve that in a later sprint

Reply:
Yes!